src/env.js

```diff
@@ -57,14 +59,13 @@ export class Env {
 
     /**
      * Creates a new instance of Env.
-     * @param {object} [source] The environment variable object to read from. 
+     * @param {any} [source] The environment variable object to read from. 
```

Comment:
Curious as to the reason for this change?

Reply:
That may be unneeded; I'll revert it